neuralpp/experiments/pimt_experiment.py

```diff
@@ -123,6 +128,13 @@ def two_normals_expectation():
 
 
 if __name__ == "__main__":
+    evaluation_general_with_result("branch pruning example",
+                                   normalization_generator(lambda: basic_integral(x, Z3SolverExpression.from_expression(x > -20.0) & (x < 20.0), normal_square)),
+                                   {})
+    evaluation_general_with_result("branch pruning example-2",
```

Comment:
The function `evaluation_general_with_result` can be used when we want to verify the result. However, it won't work for some larger examples due to some limitations of SymPy's C code generation; also for large examples, the conversion time from our library representation to a SymPy one (https://github.com/rodrigodesalvobraz/neuralpp/pull/51/files#diff-f980e5935011a0e635e5749a155ed7feae9a9ea006fd4415f48372b77dd87e86R73) could take too long.